app/services/collections/delete.rb

```diff
@@ -19,7 +19,7 @@ def call
     attr_reader :user, :collection
 
     def delete_collection
-      CollectionWebsite.where('collection_id = ?', collection.id).destroy_all
+      CollectionWebsite.where(collection_id: collection.id).destroy_all
```

Comment:
Can't this be achieved by doing `dependent: destroy` on `has_many :collection_websites`?